python/tk_maya/menu_generation.py

```diff
@@ -197,19 +200,56 @@ def _add_app_menu(self, commands_by_app):
                 cmd_obj = commands_by_app[app_name][0]
                 if not cmd_obj.favourite:
                     # skip favourites since they are already on the menu
-                    cmd_obj.add_command_to_menu(self._menu_handle)
+                    cmd_obj.add_command_to_menu(self._menu_path)
+
+
+class Callback(object):
+    def __init__(self, callback):
+        self.callback = callback
+
+    def __call__(self, *args):
```

Comment:
Accepts any args that might get thrown it's way such as the menu item state. We don't need the state but the method needs to be able to accept it.

Reply:
I feel this comment would be better as a comment in the code, to avoid someone doing some cleanup by mistake.

Reply:
Also, naming parameters `_` is a classic way to tell the reader (and linters) that a variable is unused.